internal/config/config.go

```diff
@@ -63,6 +63,8 @@ func Read(config *Configuration, configFile string) (found bool, configFileUsed
 	v.AddConfigPath("/etc")
 	v.SetConfigName("enduro")
 	v.SetDefault("api.processing", a3m.ProcessingDefault)
+	// TODO: Change from a3m.taskqueue to pres.taskqueue, and add other preservation configuration settings
```

Comment:
Why not replace `a3m.taskqueue` with `preservation.taskqueue` now? I don't understand why we'd want to have `perservationSystem` and `a3m.taskqueue` in the config.